WC_Layers/OpenMC_PhotonTransport.py

```diff
@@ -0,0 +1,138 @@
+parser.add_argument('--W_inner_radius', type=float, required=True)
+parser.add_argument('--W_outer_radius', type=float, required=True)
+parser.add_argument('--C_inner_radius', type=float, required=True)
+
+args = parser.parse_args()
+
+fp = args.filepath
+E_1 = args.element_1
+E_2 = args.element_2
+R_W_1 = args.W_inner_radius
+R_W_2 = args.W_outer_radius
+R_C_1 = args.C_inner_radius
+
+# fp = '../../ALARA/data/elelib.std'
+# E_1 = 'W'
+# E_2 = 'C'
+# R_W_1 = 1000
+# R_W_2 = 1005
+# R_C_1 = 995
+bounds = [0, 
+                 1.00e+4, 
+                 2.00e+4, 
+                 5.00e+4, 
+                 1.00e+5, 
+                 2.00e+5, 
+                 3.00e+5, 
+                 4.00e+5, 
+                 6.00e+5, 
+                 8.00e+5, 
+                 1.00e+6, 
+                 1.22e+6, 
+                 1.44e+6, 
+                 1.66e+6, 
+                 2.00e+6, 
+                 2.50e+6, 
+                 3.00e+6, 
+                 4.00e+6, 
+                 5.00e+6, 
+                 6.50e+6, 
+                 8.00e+6, 
+                 1.00e+7, 
+                 1.20e+7, 
+                 1.40e+7, 
+                 2.00e+7]
+
+#Define source:
+def make_source(W_Shell, C_Shell, Cells):
+    Source_List = []
+    Particle_Filter = openmc.ParticleFilter('photon')
+    Total_Mesh = openmc.UnstructuredMesh(Mesh_File, library='moab')
+    for index, bound in enumerate(bounds[:-1]):
+        Mesh_Dist = openmc.stats.MeshSpatial(Total_Mesh, strengths=strengths_list[index], volume_normalized=False)
+        Energy_Dist = openmc.stats.Uniform(a=bounds[index], b=bounds[index + 1])
+        #Source strengths given by strengths_list created by Source_Mesh_Reader
+        Source = Source_List.append(openmc.IndependentSource(space=Mesh_Dist, energy=Energy_Dist, strength=summed_strengths[index], particle='photon', domains=Cells))
+    return Source_List, Particle_Filter, Total_Mesh, Mesh_Dist
+
+# Define tallies
+def tallies(W_Shell, C_Shell, Particle_Filter, Total_Mesh):
+    Total_Filter = openmc.MeshFilter(Total_Mesh)
+    
+    neutron_tally = openmc.Tally(tally_id=1, name="Neutron tally")
+    neutron_tally.scores = ['flux', 'elastic', 'absorption']
+    
+    # Implementing filter for neutron tally through W shell
+    cell_filter = openmc.CellFilter([W_Shell, C_Shell])
+    neutron_tally.filters = [cell_filter, Total_Filter]
+
+    # Creating a tally to get the flux energy spectrum.
+    # An energy filter is created to assign to the flux tally.
+    energy_filter_flux = openmc.EnergyFilter.from_group_structure("VITAMIN-J-42")
+
+    spectrum_tally = openmc.Tally(tally_id=2, name="Flux spectrum")
+    # Implementing energy and cell filters for flux spectrum tally
+    spectrum_tally.filters = [cell_filter, Total_Filter, energy_filter_flux, Particle_Filter]
+    spectrum_tally.scores = ['flux']
+    
+    tall = openmc.Tallies([neutron_tally, spectrum_tally])
+    tall.export_to_xml()

```

Comment:
don't export in each function, just return the OpenMC tallies object for handling by the calling function